lib/rubycritic/rake_task.rb

```diff
@@ -0,0 +1,50 @@
+require "rake"
+require "rake/tasklib"
+
+module Rubycritic
+  class RakeTask < ::Rake::TaskLib
+    attr_accessor :name
+    attr_accessor :report
+    attr_accessor :format
+    attr_accessor :paths
+    attr_accessor :options
+
+    def initialize(*args, &task_block)
+      @name = args.shift || :rubycritic
+      @fail_on_error = true
+      @options = []
+      @format = "html"
+      @report = nil
+      @paths = []
+
+      desc "Run RubyCritic" unless ::Rake.application.last_comment
+
+      task(name, *args) do |_, task_args|
```

Comment:
You kind of completely ignored my comment from your old pull request :-1: 
So let me repeat:

I still dont think we need this (and shouldnt have it since its obscure). 
We dont do this in Reek as well. Check it out, we have an extremely "easy to understand and reason about" definition here: https://github.com/troessner/reek/blob/master/lib/reek/rake/task.rb which allows us to use Reek similar like you intend to do with RubyCritic: https://github.com/troessner/reek/blob/master/tasks/reek.rake

How about setting this up in a similar way here and drop the if branch completely?